test.http

```diff
@@ -0,0 +1,7 @@
+POST http://0.0.0.0:3000
+content-type: application/json
+X-internal-Challenge: test-secret-key
+
+{
+    "garbage": "garbage"
```

Comment:
Maaaybe a different word here 😅

Reply:
this is something to be deleted, i made this file for local testing. I took this from test files